### ISSUE TITLE
Desk dog is missing

### ISSUE THREAD

**Issue by Cateatacat** [Makssuper747/OpenMX-Add-ons#1]
The desk dog app is not in the repository

**Comment by Makssuper747** [comment]
> The desk dog app is not in the repository

Yeah I know there's still a lot of apps missing but I will be adding them, so in the future all will be available here.
